src/Link/Link.svelte

```diff
@@ -1,4 +1,10 @@
 <script>
+  /**
+   * Specify href value
+   * @type {string} [href]
+   */
+  export let href;
```

Comment:
I know it's the same, but to stay consistent initialize `href` as `undefined`

Reply:
Well, I (badly?) supposed that not initializing a prop made it kind of mandatory. Nevertheless, I will change this!